docs/release/osg-36.md

```diff
@@ -87,10 +87,14 @@ Latest News
 
 !!! danger
     The format of the HTCondor job queue log has changed. Once you have updated
-    the condor_schedd to HTCondor 10.6.0, you may only downgrade to a version
-    that can parse this new format.
+    the Access Point and HTCondor-CE (i.e., hosts with a `condor_schedd` daemon)
+    to HTCondor 10.6.0, you may only downgrade to a version that can parse this
+    new format.
     (LTS: 10.0.4 and later, feature: 10.5.0 and later)
 
+    We recommend upgrading your Access Points and HTCondor-CE hosts to 10.0.4 or
+    10.5.0 first, then proceeding with an upgrade to 10.6.0.
+
```

Comment:
It would be fine to update to the latest 10.0.x release before going to the latest 10.x.0 release. This may be easier for admins to update to from the main repo and then update from the upcoming repo. Thoughts?

Reply:
So just 10.0.6 instead of 10.0.4?

Reply:
Well, it's a moving target. I just thought for admins that don't know how to install a specific version, updating to the latest LTS works just as well.

Reply:
```suggestion
    We recommend upgrading your Access Points and HTCondor-CE hosts to the latest 10.0
    release or 10.5.0 first, then proceeding with an upgrade to 10.6.0.
```